src/core/models.py

```diff
@@ -6,6 +6,7 @@
 from django.db import models
 from django.utils.translation import gettext_lazy as _
 
+from .choices import CATEGORY_CHOICES
```

Comment:
**nitpick (non-blocking):** If you have time, please consider adding other  options group such as [LANGUAGE_CHOICES](https://github.com/pycontw/pycon.tw/blob/4f9e349caf253fd8488a96aa28af315e9a4f2964/src/core/models.py#L94), [PYTHON_LVL_CHOICES](https://github.com/pycontw/pycon.tw/blob/4f9e349caf253fd8488a96aa28af315e9a4f2964/src/core/models.py#L111), [RECORDING_POLICY_CHOICES](https://github.com/pycontw/pycon.tw/blob/4f9e349caf253fd8488a96aa28af315e9a4f2964/src/core/models.py#L111), etc. to the `chloices.py` file, thanks!